finalPrep/README.md

```diff
@@ -44,31 +43,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
-
+- [Week 1](https://user-images.githubusercontent.com/99286590/156654509-97c0e1a3-011b-4630-b588-b68f9f8ee79e.png)
```

Comment:
Looks like you're dedicating a realistic amount of time to Turing, and I'm so glad to see those personal times blocked off too!